python-package/xgboost/training.py

```diff
@@ -342,6 +369,18 @@ def cv(params, dtrain, num_boost_round=10, nfold=3, metrics=(),
                     as_pandas=as_pandas)
         results.append(res)
 
+        if early_stopping_rounds is not None:
+            score = res[0]
+            if (maximize_score and score > best_score) or \
+                    (not maximize_score and score < best_score):
+                best_score = score
+                best_score_i = i
+            elif i - best_score_i >= early_stopping_rounds:
+                sys.stderr.write("Stopping. Best iteration: {}\n".format(best_score_i))
+                best_score = best_score
```

Comment:
What does this line do? Also maybe a better naming for `best_score_i`? This `stderr.write` can be left as it is (not like the previous one).